x-pack/plugins/apm/server/routes/settings/storage_explorer/get_doc_count_per_processor_event.ts

```diff
@@ -70,6 +71,14 @@ export async function getDocCountPerProcessorEvent({
                   field: PROCESSOR_EVENT,
                   size: 10,
                 },
+                aggs: {
```

Comment:
@dgieselaar do you think it's better if the number of sampled transactions is calculated from another request instead of having a nested aggregation here?